### ISSUE TITLE
Can't create a new credential with an organization administrator account

### ISSUE THREAD

**Issue by nbruniaux** [ansible/awx#1925]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

 - UI

##### SUMMARY

Can't create a new credential with an organization administrator account

![2018-05-30 10_34_30-ansible awx _ create credential](https://user-images.githubusercontent.com/37828871/40709434-4e9de148-63f6-11e8-9892-6572b221dddc.jpg)

##### ENVIRONMENT
* AWX version: 1.0.6.9
* AWX install method: docker on linux
* Ansible version:  2.5.3
* Operating System: Windows 7 
* Web Browser: Chrome, Firefox

##### STEPS TO REPRODUCE

- Log in to AWX with an organization administrator account (no system administrator)
- Create new credentials of any type
- When saving the error message "Unexpected Error: {"detail":"You do not have permission to perform this action."}" appears

##### ADDITIONAL INFORMATION

When passing through the API or using system administrator account, there is no problem

**Comment by AlanCoding** [comment]
I cannot reproduce this.

I tried as the admin of the Default org, user's pk=4.

Request data:

```json
{"name":"org admins","credential_type":1,"inputs":{"username":"asdf","password":"asdf"},"user":4}
```

Could you inspect the network traffic and give us an example of the request data being sent on the POST request to create the credential? Additionally, does the `"user":4` part of your data match the requesting user's primary key?

**Comment by nbruniaux** [comment]
The request data being sent on the POST request to create the credential is :

{"name":"issue_1925","organization":3,"credential_type":1,"inputs":{"username":"xxxx","password":"xxxx"},"user":15}

"user": 15 is the primary key of the requesting user.

In addition, I have just found that the problem only occurs when in "Setting / System / Organization Admins Can Manage Users And Teams" is OFF.

I am using an LDAP integration for creating and managing users, I would like to keep this setting at OFF while maintaining the ability to manage credentials for organization administrators.

**Comment by AlanCoding** [comment]
Thanks for digging in! I can confirm the bug that you described. The the setting `MANAGE_ORGANIZATION_AUTH` set to False, it appears that org admins can not create credentials in their organization.

My testing shows that they also cannot create a credential if the Organization field is left blank.

From the request data, the UI has done nothing wrong, I am recategorizing as an API bug.

**Comment by ryanpetrello** [comment]
@nbruniaux we've just released 1.0.7, which we believe resolves the underlying issue here. You can try it out here: https://github.com/ansible/awx/releases/tag/1.0.7

Let us know if you're still seeing this issue after installing the latest awx - thanks!
